RFC_PROCESS.md

```diff
@@ -0,0 +1,73 @@
+# RFC Process
+
+## What's an RFC?
+
+The name itself is a reference to the IETF's Request For Comments process, and
+basically involves a document or series of documents which are drafted,
+reviewed, and eventually ratified (approved) by the Ayo.js community through
+discussion among those interested.
+
+An RFC can extend, modify, or alter any part of the Ayo.js project's code or
+documents, whether or not they've been previously documented. This includes
+proposing new features, proposing major breaking changes, changes to the Code of
+Conduct and other community policies, changes to the governance structure of the
+project, and anything else that affects a significant part of the community. It
+can also propose entirely new policies and community agreements.
+
+## How do I create an RFC?
+
+* Go to https://github.com/ayojs/ayo
+* Create a PR with a new RFC document:
+  * Must be in `rfcs`.
```

Comment:
suggestion for clarity (if my reading is correct?): "Must be in the `rfcs` folder."